test/hummingbot/connector/exchange/ascend_ex/test_ascend_ex_api_order_book_data_source.py

```diff
@@ -340,24 +328,6 @@ def test_listen_for_subscriptions_enqueues_diff_and_trade_messages(self, ws_conn
         self.assertEqual(1, diffs_queue.qsize())
         self.assertEqual(1, trade_queue.qsize())
 
-    @patch("aiohttp.client.ClientSession.ws_connect", new_callable=AsyncMock)
-    def test_listen_for_subscriptions_handle_ping_message(self, ws_connect_mock):
-        # In AscendEx Ping message is sent as a aiohttp.WSMsgType.TEXT message
-        mock_response = {"m": "ping", "hp": 3}
-        ws_connect_mock.return_value = self.mocking_assistant.create_websocket_mock()
-        self.mocking_assistant.add_websocket_aiohttp_message(
-            websocket_mock=ws_connect_mock.return_value,
-            message=json.dumps(mock_response),
-            message_type=aiohttp.WSMsgType.TEXT,
-        )
-
-        self.listening_task = self.ev_loop.create_task(self.data_source.listen_for_subscriptions())
-
-        self.mocking_assistant.run_until_all_aiohttp_messages_delivered(ws_connect_mock.return_value)
-        sent_json = self.mocking_assistant.json_messages_sent_through_websocket(ws_connect_mock.return_value)
-
-        self.assertTrue(any(["pong" in str(payload) for payload in sent_json]))
-
```

Comment:
I believe this is still required (check the comment I left in the module file)

Reply:
According to the documentation there are 2 methods of pinging. One is if the client sends pings by itself, the other is if it replies to a server's ping. Since the WSAssistant -> WSConnection -> aiohttp.ClientWebSocketResponse ping the server automatically, this first method shouldn't be required. But it may be good to keep it just in case.

![image](https://user-images.githubusercontent.com/7762229/158134899-669ab69d-da49-4f51-b435-094e10f579de.png)